frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -30,6 +19,26 @@ export const getCephPVs = (pvsData: K8sResourceKind[] = []): K8sResourceKind[] =
     );
   });
 
+export const getCephPVCs = (
+  cephSCNames: string[] = [],
+  pvcsData: K8sResourceKind[] = [],
+  pvsData: K8sResourceKind[] = [],
+): K8sResourceKind[] => {
+  const cephPVs = getCephPVs(pvsData);
+  const boundPVCs = pvcsData.filter((pvc) => _.get(pvc, 'status.phase') === 'Bound');
+  const unboundPVCs = pvcsData.filter((pvc) => _.get(pvc, 'status.phase') !== 'Bound');
```

Comment:
```suggestion
pvcsData.forEach((pvc) => (_.get(pvc, 'status.phase') === 'Bound' ? boundPVCs : unboundPVCs).push(pvc));
```

Reply:
What is wrong with having multiple lines? It does not get me a performance boost. However, it does decrease the readability of the code.

Reply:
Not multiple lines but two iterations I see.
Things can be done in a single go

Reply:
"Premature optimization is the root of all evil". Donald Knuth.

Reply:
Just to correct, not an optimization but enhancement ! 
What you have written is good but its just that I could not resist on this similar 'Bound' complimentary condition used twice.
Sorry for hurting your code.

Reply:
I could put the 'Bound' as a const or create a function `isBound` if you want me to. But using Array. prototype property on the output of a ternary operator looks cryptic to me.

Reply:
Dont do that.